pydantic_ai_slim/pydantic_ai/providers/cerebras.py

```diff
@@ -19,10 +19,19 @@
 except ImportError as _import_error:  # pragma: no cover
     raise ImportError(
         'Please install the `openai` package to use the Cerebras provider, '
-        'you can use the `openai` optional group — `pip install "pydantic-ai-slim[openai]"`'
+        'you can use the `cerebras` optional group — `pip install "pydantic-ai-slim[cerebras]"`'
     ) from _import_error
 
 
+def zai_model_profile(model_name: str) -> ModelProfile | None:
```

Comment:
Please move this to `profiles/zai.py` and for now we can just return `None`, as I think the 3 properties you set here currently are more a quality of the provider/API than the model itself, so it makes sense to  just set them in `CerebrasProvider.model_profile`